core/main.js

```diff
@@ -85,9 +85,15 @@ Object.defineProperties(Blockly, {
    */
   mainWorkspace: {
     set: function(x) {
```

Comment:
Here and for selected, the deprecation notice for set*() doesn't really make sense since it points you to the getter.

Reply:
True. The other option is we just don't include a suggested replacement in the set-case (since I don't want people to set the main workspace or selected object themselves). I thought it would be better to include the get method so that people have a little bit of info, but if it's confusing I'm more than happy to remove it.

Reply:
Fair enough, I think you're right that the context is useful. Leaving as-is SGTM.